Unity-MCP-Plugin/Assets/root/Editor/Scripts/UI/Window/MainWindowEditor.CreateGUI.cs

```diff
@@ -248,6 +249,39 @@ public void CreateGUI()
                 }
             }));
 
+            // Tools Configuration
+            // -----------------------------------------------------------------
+            var btnOpenTools = root.Query<Button>("btnOpenTools").First();
+            btnOpenTools.RegisterCallback<ClickEvent>(evt =>
+            {
+                McpToolsWindow.ShowWindow();
+            });
+
+            var toolsStatusLabel = root.Query<Label>("toolsStatusLabel").First();
+
+            McpPlugin.McpPlugin.DoAlways(plugin =>
+            {
+                Observable.Interval(TimeSpan.FromSeconds(1))
+                    .ObserveOnCurrentSynchronizationContext()
+                    .Subscribe(_ =>
+                    {
```

Comment:
No timers with heavy operations inside for updating couple of numbers.
Need to replace with `toolManager.OnToolsUpdated` IObservable to get the event efficiently when something get changed.